packages/fx-core/tests/component/generator/officeAddinGenerator.test.ts

```diff
@@ -180,6 +181,58 @@ describe("OfficeAddinGenerator", function () {
     chai.expect(inputs[OfficeHostQuestion.name]).to.eq("Outlook");
   });
 
+  it("should copy addin files and convert manifest if addin folder is specified with xml manifest", async () => {
+    const inputs: Inputs = {
+      platform: Platform.CLI,
+      projectPath: testFolder,
+      "app-name": "office-addin-test",
+    };
+    inputs["capabilities"] = ["taskpane"];
+    inputs[AddinProjectFolderQuestion.name] = "somepath";
+    inputs[AddinLanguageQuestion.name] = "TypeScript";
+    inputs[AddinProjectManifestQuestion.name] = "manifest.xml";
+
+    const copyAddinFilesStub = sinon
+      .stub(HelperMethods, "copyAddinFiles")
+      .callsFake((from: string, to: string) => {
+        return;
+      });
+    const updateManifestStub = sinon
+      .stub(HelperMethods, "updateManifest")
+      .callsFake(async (destination: string, manifestPath: string) => {
+        return;
+      });
+    const convertProjectStub = sinon
+      .stub()
```

Comment:
Is the stub() deliberately called with no input?

Reply:
The example I followed for proxyquiry did that.  I didn't try doing something different.